sdk/eventhub/Azure.Messaging.EventHubs/src/EventData.cs

```diff
@@ -139,17 +139,17 @@ internal SystemEventProperties()
             /// </summary>
             ///
             /// <param name="sequenceNumber">The logical sequence number of the event within the partition stream of the Event Hub.</param>
-            /// <param name="enqueuedTimeUtc">The date and time, in UTC, that the event was received by the partition.</param>
+            /// <param name="enqueuedTime">The date and time that the event was received by the partition.</param>
```

Comment:
Let's leave the ", in UTC, " call outs in the comments, please.

Reply:
Done.